src/components/PopupWithChangeAvatar.js

```diff
@@ -0,0 +1,8 @@
+import PopupWithForm from "./PopupWithForm.js";
+
+class PopupWithChangeAvatar extends PopupWithForm {
+  constructor(popupSelector, handleSubmit) {
+    super(popupSelector, handleSubmit);
+  }
```

Comment:
`PopupWithChangeAvatar.js` is not needed because the `avatar` modal should be created with `PopupWithForm`. It's redundant